src/test/java/com/fasterxml/jackson/databind/struct/TestUnwrappedWithJsonCreator.java

```diff
@@ -0,0 +1,128 @@
+package com.fasterxml.jackson.databind.struct;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.databind.*;
+
+/**
+ * Tests to verify [databind#1467].
+ */
+public class TestUnwrappedWithJsonCreator extends BaseMapTest
```

Comment:
Newer naming convention leads to `UnwrappedWith[Json]CreatorTest` (we haven't renamed all or even most tests but trying to move in that direction).